src/components/UI/Display/MAPI/clusters/ClusterDetail/ClusterDetailWidgetOptionalValue.tsx

```diff
@@ -40,17 +43,23 @@ const ClusterDetailWidgetOptionalValue: React.FC<IClusterDetailWidgetOptionalVal
     return <NotAvailable />;
   }
 
+  const refreshableKey = JSON.stringify(value);
```

Comment:
Silly way of serializing data structures, so `RefreshableLabel` would know if a value changed